src/bootstrap/bootstrap.py

```diff
@@ -18,6 +18,7 @@
 import sys
 import tarfile
 import tempfile
+import urllib2
```

Comment:
No the `urllib2` module is Python 2-only, considering #36509 just landed a week ago I think we should only use Py3k-compatible modules from now on.

This should be enough without introducing `six` as an additional dep:

```python
try:
    from urllib2 import urlopen
except ImportError:
    from urllib.request import urlopen
```